ui/imports/shared/stores/CommunityTokensStore.qml

```diff
@@ -42,6 +42,14 @@ QtObject {
         console.log("TODO: Deploy Asset backend!")
     }
 
+    function deleteToken(communityId, contractUniqueKey) {
+        console.log("TODO: Delete token bakend!")
+    }
+
+    function retryMintToken(communityId, contractUniqueKey) {
```

Comment:
Is tokenKey the same as contractUniqueKey?

Reply:
Yes, it will be renamed in my next pr to be always `tokenKey`